lib/generators/rolemodel/optics/base/base_generator.rb

```diff
@@ -0,0 +1,31 @@
+module Rolemodel
+  module Optics
+    class BaseGenerator < Rails::Generators::Base
+      source_root File.expand_path('templates', __dir__)
+
+      def add_optics_package
+        run 'yarn add @rolemodel/optics'
+      end
+
+      def add_slim
+        generate 'rolemodel:slim'
```

Comment:
I kind of think we could remove this part from Optics entirely. Not anything specifically that depends on slim at this point as we don't ship a slim based styleguide like we used to

Reply:
good call.  I added this reflexively, because it used to add the slim gem..